clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/controllers/OperationsController.groovy

```diff
@@ -189,7 +190,7 @@ class OperationsController {
                                      AtomicOperationConverter converter,
                                      Map descriptionInput) {
     def descriptionClass = converter.metaClass.methods.find { it.name == "convertDescription" }.returnType
-    descriptionPreProcessors.findAll { it.supports(descriptionClass) }.each {
+    descriptionPreProcessors.findAll { it.supports(descriptionClass, ImmutableMap.copyOf(descriptionInput)) }.each {
```

Comment:
here would be a good spot to count if a preProcessor took effect - passing in a base Metric Id that had tags prepopulated that includes the remote endpoint invoked, and adding in the preprocessor name as another tag?